test/libsolidity/util/TestFileParserTests.cpp

```diff
@@ -460,6 +460,20 @@ BOOST_AUTO_TEST_CASE(call_multiple_arguments_mixed_format)
 	);
 }
 
+BOOST_AUTO_TEST_CASE(call_signature_array)
+{
+	char const* source = R"(
+		// f(uint256[]) ->
+		// f(uint256[3]) ->
+		// f(uint256[3][][], uint8[9]) ->
```

Comment:
```suggestion
		// f(uint256[3][][],uint8[9]) ->
```

Actually at some point the space should actually even throw ;-D...

Reply:
Ok, since this is not the only one we might as well postpone this until we actually disallow all of that.